db/config.c

```diff
@@ -279,10 +304,42 @@ int deferred_do_commands(struct dbenv *env, char *option,
     int rc = 0;
 
     tok = segtok(option, len, &st, &tlen);
-    if (tokcmp(tok, tlen, "sqllogger") == 0)
+    if (tokcmp(tok, tlen, "sqllogger") == 0) {
         sqllogger_process_message(option + st, len - st);
-    else if (tokcmp(tok, tlen, "do") == 0)
+    } else if (tokcmp(tok, tlen, "do") == 0) {
         rc = process_command(env, option + st, len - st, 0);
+    } else if (tokcmp(tok, tlen, "comdbg") == 0) {
+        char *tablename;
+        int dbnum;
+
+        // table name
+        tok = segtok(option, len, &st, &tlen);
+        if (tok == 0) {
+            logmsg(LOGMSG_ERROR, "no table name specified for comdbg\n");
+            return -1;
+        }
+        tablename = tokdup(tok, tlen);
+
+        // dbnum (optional)
+        tok = segtok(option, len, &st, &tlen);
+        if (tlen == 0)
```

Comment:
Sorry to nitpick, but I think we should make disabling it explicit
ie:
comdbg foobar         <- syntax error
comdbg foobar 0     <- disables dbnum

Otherwise, looks great!

Reply:
done